CI jobs failing due to missing `serde` dependency in contract bindings

Consider the following jobs:

- https://github.com/cartesi/dave/actions/runs/15253392013/job/42895252780
- https://github.com/cartesi/dave/actions/runs/15253392013/job/42895252215

Both are failing with the following error:

```
[...]
error[E0433]: failed to resolve: use of unresolved module or unlinked crate `serde`
Error:     --> cartesi-rollups/contracts/bindings-rs/src/contract/daveconsensusfactorytest.rs:6528:32
     |
6528 |     #[derive(serde::Serialize, serde::Deserialize)]
     |                                ^^^^^ use of unresolved module or unlinked crate `serde`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `cartesi-dave-contracts` (lib) due to 310 previous errors
warning: build failed, waiting for other jobs to finish...
error: could not compile `cartesi-prt-contracts` (lib) due to 506 previous errors
error: Recipe `check-rust-workspace` failed on line 61 with exit code 101
Error: Process completed with exit code 101.
```

--- a/cartesi-rollups/contracts/justfile
+++ b/cartesi-rollups/contracts/justfile
@@ -45,6 +45,7 @@ clean-deployments:
 bind: clean-bindings
     forge bind --alloy --select {{BINDINGS_FILTER}} \
         --module --bindings-path {{BINDINGS_DIR}} \
+        --skip-extra-derives \
         --root {{SRC_DIR}}
 
 deploy-core *OPTS: \

--- a/prt/contracts/justfile
+++ b/prt/contracts/justfile
@@ -43,6 +43,7 @@ clean-deployments:
 bind: clean-bindings
     forge bind --alloy --select "{{BINDINGS_FILTER}}" \
     --module --bindings-path {{BINDINGS_DIR}} \
+    --skip-extra-derives \
     --root {{SRC_DIR}}
 
 deploy-core *OPTS: \

